ts/components/messages/PaymentMessageComponent.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from "react";
+
+import { Button, Icon, Left, ListItem, Right, Text, View } from "native-base";
+import { connectStyle } from "native-base-shoutem-theme";
+import mapPropsToStyleNames from "native-base/src/utils/mapPropsToStyleNames";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import I18n from "../../i18n";
+import ROUTES from "../../navigation/routes";
+import { convertDateToWordDistance } from "../../utils/convertDateToWordDistance";
+
+export type OwnProps = Readonly<{
+  createdAt: Date;
+  id: string;
+  markdown: string;
+  navigation: NavigationScreenProp<NavigationState>;
+  serviceName: string;
+  serviceOrganizationName: string;
+  serviceDepartmentName: string;
+  subject: string;
+  paymentAmount: number;
+  paymentNoticeNumber: string;
+}>;
+
+export type Props = OwnProps;
+
+/**
+ * Implements a component that show a message with payment cta in the MessagesScreen List
+ */
+class PaymentMessageComponent extends React.Component<Props> {
+  public convertAmount = (amount: number): string => {
+    return Number.isInteger(amount)
```

Comment:
try [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) instead

Reply:
it's possible use I18n eg I18n.toCurrency(amount,{unit: "€",separator: ","})
https://github.com/fnando/i18n-js#number-formatting